treeherder/model/migrations/0036_bugjobmap_internal_bug_refs.py

```diff
@@ -10,6 +11,17 @@ def set_internal_fks(apps, schema_editor):
     created internal ID and Bugzilla ID should be out of sync.
     """
     BugJobMap = apps.get_model("model", "BugJobMap")
+    Bugscache = apps.get_model("model", "Bugscache")
+    bug_ids = list(BugJobMap.objects.all().distinct("bugzilla_id").values_list("bugzilla_id", flat=True))
+    now = timezone.now(),
+    for bug_id in bug_ids:
+        if not Bugscache.objects.filter(bugzilla_id=bug_id).exists():
+            Bugscache.objects.get_or_create(
+                id=bug_id,
+                bugzilla_id=bug_id,
+                modified=now,
+                summary="(no bug data fetched)",
+            )
```

Comment:
I think we should rather let Django perform the GET from get_or_create to save one extra query per ID.
```suggestion
        Bugscache.objects.get_or_create(
            id=bug_id,
            bugzilla_id=bug_id,
            defaults={
                "modified": now,
                "summary": "(no bug data fetched)",
            }
        )
```

Reply:
Indeed it can be changed to only `bugzilla_id=bug_id` (id being set in default or automatically as it is an autoincrement field).
It should not affect this migration though as intenal/bugzilla IDs should still be synced yet.